### ISSUE TITLE
Error after upgrading to 0.62.0

### ISSUE THREAD

**Issue by librod89** [daily-co/react-native-daily-js#55]
# Expected behavior

Upgrade from v0.61.0 to v0.62.0 with no failures.

# Describe the bug (unexpected behavior)

After upgrading the `@daily-co/react-native-daily-js` version from 0.61.0 to 0.62.0, my app fails on the simulator and physical device with the following error. I made no other changes to the code.

`EventOnHostDestroy is not a supported event type for DailyNativeUtils. Supported events are: EventSystemScreenCaptureStop, EventSystemScreenCaptureStart`

# Steps to reproduce

* Start the app with `npx expo start --clear`
* The code runs `DailyIframe.createCallObject();`
* Error is thrown

# Screenshots

<img src="https://github.com/daily-co/react-native-daily-js/assets/4965672/249707cc-22d8-4e4e-bbc2-e93d56cdd7b1" width="300">

# System information 

 

* Device: iOS simulator iPhone 15 Pro Max
* OS, version: iOS 17.0
* Dependencies:
    ```"@config-plugins/react-native-webrtc": "8.0.0",
    "@daily-co/config-plugin-rn-daily-js": "0.0.5",
    "@daily-co/daily-react": "0.19.0",
    "@daily-co/react-native-daily-js": "0.62.0",
    "@daily-co/react-native-webrtc": "118.0.3-daily.2",
    "expo": "50.0.17",
    ...

**Comment by filipi87** [comment]
Hi @librod89,

I have just tried to replicate the issue with our [Expo demo](https://github.com/daily-demos/daily-expo-demo), updating it to use `v0.62.0`, but everything keeps working.

Since we had some changes in the native code, I believe you need to update your development build. You can follow the instructions that we have mentioned [here](https://github.com/daily-demos/daily-expo-demo?tab=readme-ov-file#building-locally).

Basically for local build on iOS:
```
# Before a native app can be compiled, the native source code must be generated.
npx expo prebuild
npx expo run:ios --device 
```

Let me know if that works.

**Comment by librod89** [comment]
That fixed it, thanks!
